examples/messages/inline_template.output.json

```diff
@@ -27,6 +27,10 @@
         "foo": "bar",
         "foo2": "bar2"
       }
+    },
+    "cumulus_meta": {
```

Comment:
Might it be a good idea not to re-use "cumulus_meta" in the payload?      I realize it's putting a few objects from that key into the payload and that implies a direct copy, but that feels like it might have the potential to cause confusion.

Reply:
Thanks for the comments.  Changed to cumulus_config.